src/styles/sizing.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --input-height: 38px;
+  --input-height: 36px;
```

Comment:
This seems to align with [the design spec](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/DevDot?node-id=7502%3A73814) and also matches the search input height.